.github/workflows/updatecli.yml

```diff
@@ -87,3 +87,6 @@ jobs:
           updatecli apply --clean --values updatecli/values.d/values.yaml \
                                   --values updatecli/values.d/versions.yaml \
                                   --config updatecli/updatecli.d/update-wins-system-agent/
+          updatecli apply --clean --values updatecli/values.d/values.yaml   \
+                                  --values updatecli/values.d/versions.yaml \
+                                  --config updatecli/updatecli.d/update-harvester-docker-machine/
```

Comment:
We could remove multiple `updatecli apply` calls and loop through targets instead:

```bash
targets=("harvester-docker-machine" "update-wins-system-agent")

for target in "${targets[@]}"; do
  updatecli apply --clean --values updatecli/values.d/values.yaml   \
                                  --values updatecli/values.d/versions.yaml \
                                  --config "updatecli/updatecli.d/${target}/"
done
```

Reply:
Is there any reason to not use --config updatecli.d?